build.gradle

```diff
@@ -7,7 +7,7 @@ apply plugin: 'io.spring.dependency-management'
 apply plugin: 'org.springframework.boot'
 
 group 'com.jinx.learning'
-mainClassName = 'com.robin.hoteless.Application'
+mainClassName = 'com.robin.hoteless.ProgramEntryPoint'
```

Comment:
Yeah, I'll bet that the name of this class is the problem; I think Spring "autoMagically" looks for the ``Application`` class for the boot up and access.